sources/mma/mma.f90

```diff
@@ -78,7 +82,8 @@ module mma
 
      !> Interface for updating the MMA
      generic, public :: update => mma_update_cpu, mma_update_vector
-     procedure, pass(this) :: mma_update_cpu
+     ! for debugging I'm making this public
```

Comment:
Are you done debugging so we can revert this?

Reply:
switched back